genbindings.sh

```diff
@@ -3,6 +3,12 @@
 set -e
 set -x
 
+if [ ! -d "$1/lightning" ]; then
+	echo "USAGE: $0 path-to-rust-lightning"
+	exit 1
+fi
+LIGHTNING_PATH="$(realpath "$1/lightning")"
```

Comment:
Looks like `realpath` doesn't exist on macos.

Reply:
Pushed a version with `readlink` does that work? Otherwise may need a cd && `$PWD` tracker.

Reply:
Getting:

```term
readlink: illegal option -- f
usage: readlink [-n] [file ...]
```

Reply:
Ok, I switched it to cd and resolve with pwd.